Code/Core/CoreTest/Tests/TestFileIO.cpp

```diff
@@ -150,7 +150,7 @@ void TestFileIO::FileCopySymlink() const
         // by the file copy API.  Also on Windows, it would make unit
         // tests require administrator privileges.
     #elif defined ( __LINUX__ )
-        AStackString<> symlinkTarget( "symlink" );
+        AStackString<> symlinkTarget( "/symlink" );
```

Comment:
My changes don't alter a link that is already an absolute path, so making the link target an absolute path so the unit test will pass.  If the target is just "symlink", then Copy() will fix it to an absolute path, and it will no longer be identical.  

Because the source and destination are in the same directory, the old behavior was also correct, and Copy() could be made smart enough to detect this (At which point I'd use PathUtils to make a relative path), but I KISS for now.